sonar-db/src/main/java/org/sonar/db/version/DatabaseVersion.java

```diff
@@ -37,7 +37,7 @@
    * versions must be previously upgraded to LTS version.
    * Note that the value can't be less than current LTS version.
    */
-  public static final int MIN_UPGRADE_VERSION = 600;
+  public static final int MIN_UPGRADE_VERSION = 1152;
```

Comment:
because 1152 ensures that all migrations of 5.6 has been applied. Seems better to me but I didn't consider 1150 at all. Maybe you have an argument to use it.

It's not 1153 because 1153 has been added with 5.6.1.